appspec.yml.deploy

```diff
@@ -10,13 +10,15 @@ hooks:
   AfterInstall:
     - location: deploy_scripts/set_permissions.sh
       timeout: 30
-    - location: deploy_scripts/install_dependencies.sh
+    - location: deploy_scripts/install_dependencies_django.sh
       timeout: 300
-    - location: deploy_scripts/run_management_commands.sh
+    - location: deploy_scripts/run_management_commands_django.sh
       timeout: 300
   ApplicationStart:
-    - location: deploy_scripts/application_start.sh
+    - location: deploy_scripts/restart_httpd.sh
+      timeout: 20
+    - location: deploy_scripts/start_crond.sh
       timeout: 20
   ValidateService:
-    - location: deploy_scripts/validate_service.sh
+    - location: deploy_scripts/curl_index.sh
```

Comment:
I believe you could CURL the status endpoint? That's a slightly stronger test because it checks the various system components.